docs/sources/reference/commandline/cli.md

```diff
@@ -2012,6 +2012,11 @@ more details on finding shared images from the command line.
 
       --help=false       Print usage
 
+The `docker stats` command will listen for a stats stream for all created containers
+requested. It will however only display data for containers when they are running.
```

Comment:
Simplify/clarify:

The `docker stats` command listens for a stats stream for all the requested created containers. However, data is only displayed for running containers.